### ISSUE TITLE
`getCurrentScene` is not a function in Pannellum

### ISSUE THREAD

**Issue by ripplearc** [hoaiduyit/react-pannellum#24]
I couldn't find `getCurrentScene` in Pannellum, which only gets `getScene`
```javascript
/**
 * Get ID of current scene.
 * @memberof Viewer
 * @instance
 * @returns {string} ID of current scene
 */
this.getScene = function() {
    return config.scene;
};
```

**Comment by ripplearc** [comment]
I have to use `getConfig().scene` from your library

**Comment by hoaiduyit** [comment]
Hi @ripplearc 

Thank for your report. I will update ASAP.

**Comment by ripplearc** [comment]
Thank you very much! 👍

**Comment by hoaiduyit** [comment]
Hi @ripplearc,

This error has been fixed in new version: `0.1.12`
Please install new version and `getCurrentScene` again, it should work now.

Thanks.
